gregoriotex/features.html

```diff
@@ -55,52 +55,91 @@
     title = 'GregorioTeX';
 </script>
 
-<h1>Features of GregorioTeX</h1>
+<h1>Features of Gregorio<span class="tex">T<span class="epsilon">e</span>X</span></h1>
 
-<h2 id="symbols">Red lines</h2>
+Gregorio<span class="tex">T<span class="epsilon">e</span>X</span> contains several features which allow you to manipulate the way your scores appear.  This page introduces you to a few of the most commonly used ones.  For a complete list, please consult GregorioRef-###.pdf (where ### is the version number) which you can find in the directory where you installed Gregorio.
 
-<p>For liturgical books, it is often useful to get red staff lines. GregorioTeX offers this possibility. To get it, you have to include the package <code>color</code>, distributed with the base TeXLive. Include it simply in the header of your main file: <code>\usepackage{color}</code>.</p>
+<h2 id="symbols">Red lines</h2>
 
-<p>To get red lines, simply call the macro <code>\redlines</code>, without argument. This will make the lines of all following scores red. To cancel it, call <code>\normallines</code>.</p>
+<p>For liturgical books, it is often useful to get red staff lines. Gregorio<span class="tex">T<span class="epsilon">e</span>X</span> offers this possibility.  To get red lines, simply call the macro <code>\gresetlinescolor{gregoriocolor}</code>, without argument. This will make the lines of all following scores red. To cancel it, call <code>\gresetlinescolor{black}</code>.</p>
```

Comment:
`without argument` should be removed since you are now, in fact, passing an argument.